src/runtime_src/core/emulation/hw_em/generic_pcie_hal2/shim.h

```diff
@@ -118,7 +118,8 @@ using addr_type = uint64_t;
       int xclLoadXclBin(const xclBin *buffer);
       //int xclLoadBitstream(const char *fileName);
       int xclLoadBitstreamWorker(char* zipFile, size_t zipFileSize, char* xmlfile, size_t xmlFileSize,
-                                 char* debugFile, size_t debugFileSize, char* memTopology, size_t memTopologySize, char* pdi, size_t pdiSize);
+                                 char* debugFile, size_t debugFileSize, char* memTopology, size_t memTopologySize,
+                                 char* pdi, size_t pdiSize, char* emuData, size_t emuDataSize);
```

Comment:
Should the pointers be const?

Reply:
Hi Sonal, discussed with Vamsi and moved all the args to struct and pass that struct as an argument.